test/com/linkedin/drelephant/util/InfoExtractorTest.java

```diff
@@ -147,4 +152,62 @@ public void testGetSchedulerInstanceNull() {
     assertEquals(null, scheduler);
```

Comment:
`assertNull` would be more obvious, I think, but I tried to limit the scope of the changes in this PR.